python_modules/dagster/dagster/core/storage/pipeline_run.py

```diff
@@ -484,6 +484,8 @@ class RunRecord(NamedTuple):
     pipeline_run: PipelineRun
     create_timestamp: datetime
     update_timestamp: datetime
+    start_time: Optional[float]
```

Comment:
maybe need to add some comments in here, indicating when these will be populated?